src/Behat/Behat/Transformation/Transformation/RuntimeTransformation.php

```diff
@@ -8,14 +8,16 @@
  * file that was distributed with this source code.
  */
 
-namespace Behat\Behat\Transformation\Call;
+namespace Behat\Behat\Transformation\Transformation;
 
 use Behat\Behat\Transformation\Transformation;
 use Behat\Testwork\Call\RuntimeCallee;
 
 /**
  * Transformation that is created and executed in the runtime.
  *
+ * @deprecated Will be removed in 4.0. Use specific transformations instead
```

Comment:
I suggest using the ``@trigger_error('...', E_USER_DEPRECATED)`` convention used by Symfony (which reminds me that someone wrote a Behat extension to collect deprecations, which I said I would review and I forgot to do it)